FIX: *Psychopy* installation (#232)
TheAxonLab/hcph-sops: docs/data-collection/preliminary.md

#### Stimuli presentation with *psychopy*

> - [ ] Log on *{{ secrets.hosts.psychopy | default("███") }}* with the username *{{ secrets.login.username_psychopy| default("███") }}* and password `{{ secrets.login.password_psychopy| default("*****") }}`.

Reviewer: good catch, you can't clone at the login screen :D

    ```
- [ ] Install bdist_mpkg, py2app and attrdict:
    ```
    pip3 install attrdict py2app bdist_mpkg
    ```

> - [ ] Install wxPython. Careful to replace the ubuntu version with the one corresponding to your system (you can check which Ubuntu version is installed on your system by running `lsb_release -a` in the terminal)

Reviewer: ```suggestion
- [ ] Install *wxPython*.
```

>     ```

Reviewer: ```suggestion
    ``` shell
```

    pip3 install -U \

>         -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-20.04 \

Reviewer: I'm sure we will forget replacing the version. Let's not allow that degree of freedom.

```suggestion
        -f https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-$( lsb_release -r -s ) \
```

        wxPython
    ```

> - [ ] Install Psychopy using the following command:

Reviewer: ```suggestion
- [ ] Install *Psychopy* using the following command:
```

    ```
    pip3 install -e .
    ```

> - [ ] Open Psychopy, open the experiment-files corresponding to each task by typing the following command in the terminal:

Reviewer: ```suggestion
- [ ] Open *Psychopy* and (optionally) a experiment file corresponding to a task by typing the following command in the terminal:
```

    ```
    psychopy your_experiment.psyexp
    ```
- [ ] For each task, check the following:
    - [ ] {{ settings.psychopy.tasks.func_qct }} (positive-control task, QCT) :

>         - [ ] time it to confirm the length, and

Reviewer: Add link to the timings in the intro?
```suggestion
        - [ ] time it to [confirm the length](intro.md#task-timing), and
```
